CHANGELOG.md

```diff
@@ -11,42 +11,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
--   Disabled "@typescript-eslint/non-nullable-type-assertion-style" rule
+- Disabled "@typescript-eslint/non-nullable-type-assertion-style" rule
```

Comment:
Could you please modify this to say that it was upgraded to eslint 9 and flat configs?

Reply:
ah yes of course, hehe the prettier tricked the pipeline a little into thinking there was a change...